exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceExporter.java

```diff
@@ -15,134 +15,86 @@
  */
 package com.google.cloud.opentelemetry.trace;
 
-import static com.google.api.client.util.Preconditions.checkNotNull;
-
-import com.google.api.gax.core.FixedCredentialsProvider;
-import com.google.api.gax.core.NoCredentialsProvider;
-import com.google.api.gax.grpc.GrpcTransportChannel;
-import com.google.api.gax.rpc.FixedTransportChannelProvider;
-import com.google.api.gax.rpc.HeaderProvider;
-import com.google.auth.Credentials;
-import com.google.auth.oauth2.GoogleCredentials;
-import com.google.cloud.trace.v2.TraceServiceClient;
-import com.google.cloud.trace.v2.TraceServiceSettings;
-import com.google.cloud.trace.v2.stub.TraceServiceStub;
-import com.google.common.collect.ImmutableMap;
-import com.google.devtools.cloudtrace.v2.AttributeValue;
-import com.google.devtools.cloudtrace.v2.ProjectName;
-import com.google.devtools.cloudtrace.v2.Span;
-import io.grpc.ManagedChannelBuilder;
+import com.google.cloud.ServiceOptions;
+import com.google.common.base.Suppliers;
 import io.opentelemetry.sdk.common.CompletableResultCode;
 import io.opentelemetry.sdk.trace.data.SpanData;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Collection;
-import java.util.List;
-import java.util.Map;
+import java.util.function.Supplier;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class TraceExporter implements SpanExporter {
 
-  private final CloudTraceClient cloudTraceClient;
-  private final ProjectName projectName;
-  private final String projectId;
-  private final TraceTranslator translator;
-
-  private static final Map<String, String> HEADERS =
-      Map.of("User-Agent", "opentelemetry-operations-java/" + TraceVersions.EXPORTER_VERSION);
-  private static final HeaderProvider HEADER_PROVIDER = () -> HEADERS;
-
-  public static SpanExporter createWithDefaultConfiguration() throws IOException {
-    TraceConfiguration configuration = TraceConfiguration.builder().build();
-    return TraceExporter.createWithConfiguration(configuration);
+  private static final Logger logger = LoggerFactory.getLogger(TraceExporter.class);
+
+  private final Supplier<SpanExporter> internalTraceExporterSupplier;
+
+  private TraceExporter(TraceConfiguration configuration) {
+    this.internalTraceExporterSupplier =
+        Suppliers.memoize(
+            () -> {
+              try {
+                return InternalTraceExporter.createWithConfiguration(configuration);
+              } catch (IOException e) {
+                logger.warn(
+                    "Unable to initialize TraceExporter. Export operation failed, switching to NoopSpanExporter",
```

Comment:
Add some more context here if you can.

"Unable to initialize Google Cloud TraceExporter" e.g.